ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -113,6 +138,20 @@ open class CommandBar: UIView {
         updateShadow()
     }
 
+    // MARK: - TokenizedControl
+
+    public typealias TokenSetKeyType = CommandBarTokenSet.Tokens
+    public var tokenSet: CommandBarTokenSet
+
+    var tokenSetSink: AnyCancellable?
```

Comment:
nit: Internal properties should placed be in between the public and private members